.github/workflows/test.yml

```diff
@@ -89,3 +89,12 @@ jobs:
       uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
     - name: Run Test
       run: go test -v ./...
+  mariner2:
+    runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
Add a TODO comment explaining why we continue on error here and saying that this line should be removed once mariner2 is fully supported.